src/libraries/System.Collections.Immutable/src/System/Linq/ImmutableArrayExtensions.cs

```diff
@@ -614,6 +641,7 @@ public static T Single<T>(this ImmutableArray<T> immutableArray, Func<T, bool> p
         /// </summary>
         /// <typeparam name="T">The type of element contained by the collection.</typeparam>
         /// <param name="immutableArray"></param>
+        /// <exception cref="ThrowNullRefIfNotInitialized">Thrown if the collection is empty.</exception>
```

Comment:
The CI failed in all legs due to the following error:

```
Error CS1574: XML comment has cref attribute 'ThrowNullRefIfNotInitialized' that could not be resolved.
```

That is because `ThrowNullRefIfNotInitialized` is not an exception, it's a method that throws an exception. You'll have to inspect that method to see what exception it throws, and use that identifier for the cref.